websites/U/uwowocosplay/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Uwowocosplay",
+	"description": {
+		"en": "Uwowo is a professional cosplay costume brand focusing on the high quality anime and game costumes and wigs with affordable price."
+	},
+	"url": "uwowocosplay.com",
+	"matches": [
+		"*://uwowocosplay.com/*"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/U2EMMgC.jpeg",
+	"thumbnail": "https://i.imgur.com/6091d7r.png",
```

Comment:
This image is currently giving a 404. This is a known issue with Imgur. If this persists, consider uploading the image to another image host.

Reply:
Yea thumbnail seems to be failing for me, logo seems fine.
Please try uploading to a different host, or I don't know if you were logged into Imgur when uploading? Because I think they auto-delete real fast if you aren't logged in.